src/Logging/src/DynamicLogger/Steeltoe.Logging.DynamicLogger.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net8.0</TargetFrameworks>
-    <Description>Steeltoe Dynamic Console Logger</Description>
+    <Description>Console logger that supports logging configuration changes while an app is running.</Description>
```

```suggestion
    <Description>Manage minimum logging levels at runtime using ConsoleLogger.</Description>
```

I regret not renaming this package to `Steeltoe.Logging.DynamicConsole` earlier. Do you think we can still do that?